hschain-utxo-lang/src/Hschain/Utxo/Lang/Sigma/DLog.hs

```diff
@@ -0,0 +1,84 @@
+-- | Proof of discrete logarithm
```

Comment:
First of all it's not really a documentation. What is "proof of logarithm"? Purpose of documentation is explain what is this thing and why it's needed. This comment fails at it completely.